pmpro-network.php

```diff
@@ -130,7 +140,7 @@ function pmpron_pmpro_checkout_boxes()
                     {
                         ?>
                         <div class="pmpro_checkout-field">
-                            <p>You will be reclaiming your site <strong><?php echo $blogname;?></strong>.</p>
+                            <p><?php echo sprintf( __( 'You will be reclaiming your site <strong>%s</strong>.', 'pmpro-network' ), $blogname ); ?></p>

```

Comment:
This tweak is tricky because we are using HTML in the translated text, so `esc_html()` won't do what we hope here without addressing that aspect first